project/AppDependencies.scala

```diff
@@ -20,8 +20,8 @@ import sbt._
 
 object AppDependencies {
 
-  private val bootstrapBackendPlay28Version = "6.4.0"
-  private val hmrcMongoPlay28Version = "0.68.0"
+  private val bootstrapBackendPlay28Version = "7.0.0"
+  private val hmrcMongoPlay28Version = "0.70.0"
 
```

Comment:
it seems there are new minor versions:
<img width="778" alt="image" src="https://user-images.githubusercontent.com/108010791/186453857-d13c9071-332f-4c17-ade4-3571e6dc8aeb.png">

Reply:
Updated.